### DISCUSSION TITLE
Is there are way to model if for all ....

### DISCUSSION THREAD

**Discussion by thomas-rudlof-bl** [openfga/community#400] [Q&A] [answered]
Assume you have a recording of a meeting  and an employee shall be able to view this resource, if he/she was a participant of the meeting or is the boss of **all** those employees. How do you model this in OpenFGA?

Obviously, this is wrong:

```fga
type employee
  relations
    define boss: [employee]
 
type recording_of_meeting
  relations
    define participant_of_meeting: [employee]
    define can_access: participant_of_meetingor or boss from participant_of_meeting
```

This models means that anyone, who is the boss of at least **some** participant can access the recording, right?

**Comment by aaguiarz** [comment, marked as answer]
@thomas-rudlof-bl great idea, I added it to the roadmap

https://github.com/openfga/roadmap/issues/71

We got someone else asking for the same feature the same day :)
